libraries/botbuilder-dialogs/src/componentDialog.ts

```diff
@@ -34,23 +34,23 @@ const PERSISTED_DIALOG_STATE = 'dialogs';
  *
  *         // Add control flow dialogs
  *         this.addDialog(new WaterfallDialog('start', [
- *             async (step) => {
```

Comment:
This change isn't needed. WaterFallDialogs are still step based so just revert this change